diagnostic.md

```diff
@@ -7,36 +7,36 @@ What command do we use to do that? Use the **correct** URL from your fork of
 this repository in your answer.
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+<!-- git clone git@github.com:aneela06/git-basics-diagnostic.git -->
 ```
 
 2.What do you do after cloning a repository, before you start making any
 changes/additions?
 
-<!-- Remove this comment and place your answer here. -->
+<!-- Before making any changes we change the branch name. -->
 
 3.What command do we use to create a new branch? Name this branch `response`
     in your answer. Then, how do we switch to that branch?
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+<!-- git checkout response then git -b response or you can just do git checkout -b response. -->
```

Comment:
The former is incorrect.  The command to create a new branch called "response" is: git branch response.  To switch to the "response" branch, you need to run: git checkout response.  The latter is correct.  "git checkout -b response" is a shorthand way to running both "git branch response" and "git checkout response" with one command.